### ISSUE TITLE
min-max value selector

### ISSUE THREAD

**Issue by weiszg** [CSTIB-Hotel/OTCAnalyser#29]
defining minimum and maximum prices should be optional. I suggest checkboxs to go with them. Also, prices can't be negative so these shouldn't go below zero (should be an easy fix - there is a minvalue property).

**Comment by waiwaing** [comment]
The database search makes an assumption that if max == min, the user doesn't care about the price, so it is in fact optional.

**Comment by weiszg** [comment]
I didn't know this => the user won't know this.

**Comment by sztankatt** [comment]
I think its fixed, as there is an (optional) keyword in the GUI. @alistairfisher ?

**Comment by alistairfisher** [comment]
Yes, I'm still not completely happy with the min/max selectors (they both show 0 by default, which is potentially confusing), so I've left this open for now.

**Comment by sztankatt** [comment]
Ok, I have changed it, so that the spinner can't be negative. by 8defdd15eb2e3a9a6c9cd599ead46af35d90274c
